server/index.js

```diff
@@ -0,0 +1,629 @@
+// TEMPORARY FILE WHILE WE MIGRATE
```

Comment:
This is a clone of the `server.js` file with a few minor changes. We'll slowly pick away at this until it's all been migrated!